src/ParseServer.js

```diff
@@ -210,7 +211,16 @@ class ParseServer {
 
     Config.validate(AppCache.get(appId));
     this.config = AppCache.get(appId);
-    hooksController.load();
+    if (hooksRefreshIntervalMs > 0) {
+      (function repeat() {
+        setTimeout(() => {
```

Comment:
@drew-gross That's a good idea, It would mean that if you used a shared cache adapter (memcache/Redis) you could invalidate the hook on any machine and they would all get the new updated hook.